src/System.Security.Cryptography.Xml/src/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard;
+      net462
```

Comment:
Other examples of this pattern say `netfx-Windows_NT;`. Do you have an example of net462 that you're basing this off of?

I don't think that any API has changed in this class family since ~net40; so we don't need to be super high on our targeting.